src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -353,3 +353,35 @@ databaseChangeLog:
                   name: closed_issue_count
                   type: bigint
                   remarks: The number of existing issues closed by this request (if it was successfully processed at all).
+  - changeSet:
+      id: users-table
+      author: tim.best@usds.dhs.gov
+      comment: information about the users of the case-issue-api
+      changes:
+        - createTable:
```

Comment:
Once we create the table, we should populate it and fix the existing `createdBy/updatedBy` records, I think. Probably need to run it twice in theory, but our user traffic level is low enough that we can probably get away with just doing it once, before we deploy the code that relies on this.

Reply:
Is there a way to look up a users "printName" from their uuid? I was figuring we would wait until everyone logs in for the database to populate. Added this commit to handle null User records https://github.com/usds/case-issue-api/pull/63/commits/350560a152125df5f04bdb458834b3698cdd728f

What is the proposed fix to the `createdBy/updatedBy` records? it was my understanding that we were going to leave these as string columns with the users uuid for now

Reply:
I believe that most of those fields in production are either `null` or in the format `${UUID};${REAL_NAME}`. So my proposal would be to do something roughly along the lines of 

```sql
    WITH all_the_actions as (
        select created_by as user_field, created_at as action_date FROM trouble_case 
        UNION ALL
        select updated_by as user_field, updated_at as action_date FROM trouble_case 
        // and so forth
    ) 
    INSERT into user_information(user_id, print_name, last_seen)
    SELECT substring(USER_FIELD, 0, 18) as UUID_PART, substring(USER_FIELD, 18) as NAME_PART, last_action_date
    FROM (select user_field, max(action_date) as last_action_date from all_the_actions);

    UPDATE trouble_case 
    SET created_by = substring(created_by, 18) 
    WHERE created_by like '%;%';
```

(the substring calls are probably invalid syntax in addition to being insufficiently paranoid, this is just by way of sketch.)

Reply:
88e73ee9000847efa8978f9d1380f4fc71e153be implements this change (correctly!)